### PR TITLE
Translated 2020 Markup in Dutch

### PR DESCRIPTION
**Pull request by noah-vdv**
Makes progress on #1750

### CODE REVIEW

**Comment by noah-vdv** [comment]
Well that wasn't supposed to go in this PR but oh well.
I found a typo on the 2020 CMS chapter where it said 29.61% but on the image it says 39.61%

File: src/content/nl/2020/markup.md [hunk 1]
```diff
@@ -0,0 +1,718 @@
...
+
+<p class="note">
+  In dit Opmaak-hoofdstuk concentreren we ons bijna uitsluitend op HTML, in plaats van SVG of MathML, die ook als opmaaktalen worden beschouwd. Tenzij anders vermeld, verwijzen de statistieken in dit hoofdstuk naar de set mobiele pagina's. Bovendien zijn de gegevens voor alle Web Almanac-hoofdstukken open en beschikbaar. Bekijk <a href="https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/">de resultaten</a> en <a href="https://discuss.httparchive.org/t/2039">deel uw observaties</a> met de community!
+</p>
+
+## Algemeen
+
+In dit gedeelte behandelen we de aspecten van een hoger niveau van HTML, zoals documenttypen, de grootte van documenten en het gebruik van opmerkingen en scripts. "Living HTML" is springlevend!
```
Comment (bazzadp) [inline src/content/nl/2020/markup.md:39]: Should we do anything about "Living HTML"? Leave as is (but wrap in spans)? OR change to "levende" that you used in featured stat above?

File: src/content/nl/2020/markup.md [hunk 2]
```diff
@@ -0,0 +1,718 @@
...
+
+Toch ontdekten we bij de bovenstaande percentielextremen dat webpagina's ongeveer 6 voorwaardelijke opmerkingen in het 90e percentiel gebruiken en 1 opmerking in het 10e percentiel. De meeste pagina's bevatten ze voor helpers zoals html5shiv, selectivizr en respond.js. Hoewel het fatsoenlijke en nog steeds actieve pagina's zijn, is onze conclusie dat veel van hen verouderde CMS-thema's gebruikten.
+
+Voor productie worden HTML-opmerkingen meestal verwijderd door build-tools. Gezien alle bovenstaande tellingen en percentages, en verwijzend naar het gebruik van opmerkingen in het algemeen, veronderstellen we dat veel pagina's worden weergegeven zonder tussenkomst van een HTML-minifier.
+
+### Script gebruik
+
+Zoals getoond in de [Top-elementen](#top-elements) sectie hieronder, is het `script` -element het 6e meest gebruikte HTML-element. Voor de doeleinden van dit hoofdstuk waren we geïnteresseerd in de manieren waarop het `script`-element wordt gebruikt op deze miljoenen pagina's uit de dataset.
```
Comment (bazzadp) [inline src/content/nl/2020/markup.md:145]: You changed the title so need to change the link. Also is it "Topelemented" ot "Top-elemented"?

```suggestion
Zoals getoond in de [Topelementen](#topelementen) sectie hieronder, is het `script` -element het 6e meest gebruikte HTML-element. Voor de doeleinden van dit hoofdstuk waren we geïnteresseerd in de manieren waarop het `script`-element wordt gebruikt op deze miljoenen pagina's uit de dataset.
```

File: src/content/nl/2020/markup.md [hunk 3]
```diff
@@ -0,0 +1,718 @@
...
+
+De laatste twee hebben natuurlijk nooit deel uitgemaakt van HTML en mogen niet worden gebruikt.
+
+## Attributen
+
+Deze sectie gaat over hoe attributen worden gebruikt in documenten en onderzoekt patronen in `data-*` gebruik. Onze bevindingen laten zien dat `class` de koningin van alle attributen is.
+
+### Topattributen
+
+Vergelijkbaar met de sectie over de meest [populaire elementen](#top-elements), gaat deze sectie in op de meest populaire attributen op internet. Gezien hoe belangrijk het `href`-attribuut is voor het web zelf, of het `alt`-attribuut om informatie [toegankelijk](./accessibility) te maken, zouden dit dan de meest populaire attributen zijn?
```
Comment (bazzadp) [inline src/content/nl/2020/markup.md:463]:
```suggestion
Vergelijkbaar met de sectie over de meest [populaire elementen](#topelementen), gaat deze sectie in op de meest populaire attributen op internet. Gezien hoe belangrijk het `href`-attribuut is voor het web zelf, of het `alt`-attribuut om informatie [toegankelijk](./accessibility) te maken, zouden dit dan de meest populaire attributen zijn?
```

File: src/content/nl/2020/markup.md [hunk 4]
```diff
@@ -0,0 +1,718 @@
...
+  content="3,97%",
+  classes="big-number",
+  sheets_gid="1981441894",
+  sql_file="summary_pages_by_device_and_doctype.sql"
+) }}
+
+Minder pagina's komen terecht in de eigenaardighedenmodus. In 2016 bedroeg dat aantal [ongeveer 7,4%](https://discuss.httparchive.org/t/how-many-and-which-pages-are-in-quirks-mode/777). Eind 2019 hebben we [4.85%](https://twitter.com/zcorpan/status/1205242913908838400) waargenomen. En nu zitten we op ongeveer 3,97%. Deze trend, om [Simon Pieters](./contributors#zcorpan) te parafraseren in zijn bespreking van dit hoofdstuk, lijkt duidelijk en bemoedigend.
+
+Hoewel we geen historische gegevens hebben om het volledige ontwikkelingsbeeld te schetsen, heeft "betekenisloze" `div`, `span` en `i` opmaak vrijwel de `table` opmaak die we hebben waargenomen in de jaren 1990 en vroege jaren 2000 [vervangen](#top-elements). Hoewel men zich kan afvragen of `div`- en `span`-elementen altijd worden gebruikt zonder dat er een semantisch geschikter alternatief is, verdienen deze elementen nog steeds de voorkeur boven `table`-markup, aangezien deze tijdens de hoogtijdagen van het oude web schijnbaar voor alles behalve tabelgegevens werden gebruikt.
```
Comment (bazzadp) [inline src/content/nl/2020/markup.md:704]:
```suggestion
Hoewel we geen historische gegevens hebben om het volledige ontwikkelingsbeeld te schetsen, heeft "betekenisloze" `div`, `span` en `i` opmaak vrijwel de `table` opmaak die we hebben waargenomen in de jaren 1990 en vroege jaren 2000 [vervangen](#topelementen). Hoewel men zich kan afvragen of `div`- en `span`-elementen altijd worden gebruikt zonder dat er een semantisch geschikter alternatief is, verdienen deze elementen nog steeds de voorkeur boven `table`-markup, aangezien deze tijdens de hoogtijdagen van het oude web schijnbaar voor alles behalve tabelgegevens werden gebruikt.
```

File: src/content/nl/2020/markup.md [hunk 5]
```diff
@@ -0,0 +1,718 @@
...
+
+Elementen per pagina en elementtypes per pagina bleven ongeveer hetzelfde en toonden [geen significante verandering](#element-diversity) in onze HTML-schrijfpraktijk in vergelijking met 2019. Dergelijke veranderingen kunnen meer tijd vergen om zich te manifesteren.
```
Comment (bazzadp) [inline src/content/nl/2020/markup.md:706]:
```suggestion
Elementen per pagina en elementtypes per pagina bleven ongeveer hetzelfde en toonden [geen significante verandering](#element-diversiteit) in onze HTML-schrijfpraktijk in vergelijking met 2019. Dergelijke veranderingen kunnen meer tijd vergen om zich te manifesteren.
```

File: src/content/nl/2020/markup.md [hunk 6]
```diff
@@ -0,0 +1,718 @@
...
+
+Gepatenteerde productspecifieke elementen zoals `g:plusone` (gebruikt op 17.607 pagina's in het mobiele voorbeeld) en `fb:like` (11.335) zijn bijna verdwenen nadat ze nog steeds [een van de meest populaire](../2019/markup#products-and-libraries-and-their-custom-markup) waren vorig jaar. We zien echter meer [aangepaste elementen](#custom-elements) voor zaken als Slider Revolution, AMP en Angular. Elementen als `ym-maatregel`, `jdiv` en `ymaps` komen ook nog steeds voor. Wat we ons voorstellen dat we hier zien, is dat, onder de zee van langzaam veranderende praktijken, HTML erg wordt ontwikkeld en onderhouden, terwijl auteurs verouderde markeringen weggooien en nieuwe oplossingen omarmen.
```
Comment (bazzadp) [inline src/content/nl/2020/markup.md:708]:
```suggestion
Gepatenteerde productspecifieke elementen zoals `g:plusone` (gebruikt op 17.607 pagina's in het mobiele voorbeeld) en `fb:like` (11.335) zijn bijna verdwenen nadat ze nog steeds [een van de meest populaire](../2019/markup#products-and-libraries-and-their-custom-markup) waren vorig jaar. We zien echter meer [aangepaste elementen](#aangepaste-elementen) voor zaken als Slider Revolution, AMP en Angular. Elementen als `ym-maatregel`, `jdiv` en `ymaps` komen ook nog steeds voor. Wat we ons voorstellen dat we hier zien, is dat, onder de zee van langzaam veranderende praktijken, HTML erg wordt ontwikkeld en onderhouden, terwijl auteurs verouderde markeringen weggooien en nieuwe oplossingen omarmen.
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Looks pretty good @noah-vdv ! Spotted a few things though.

Also looks like a few TODOs still to clean up in this chapter so should ahve finished those out before putting it forward for translation. Will try to look at those, this week before anyone else translates it, and might come back to you on any changes.

File: src/content/nl/2020/markup.md (hunk 1 above)
Comment (noah-vdv) [reply src/content/nl/2020/markup.md:39]: Yeah with that one I didn't know if it was a name like Web Almanac but if it's not then it should indeed be levende

**Review by bazzadp — APPROVED** [review]

**Pull request merged**